README.rst

```diff
@@ -24,11 +24,17 @@ It advocates a strong adherence to the guidance by restricting the range of oper
 The default theme ``gptheme`` should accommodate most use cases.
 However, the ``Theme`` object allows development of custom themes, where other formatting is required.
 
-``gptables`` is developed and maintained by the `Analysis Function`_.
+``gptables`` is developed and maintained by the `Analysis Function`_. It is
+available from `PyPI`_ or `GitHub`_. Users may also be interested in `a11ytables`_,
```

Comment:
change to: can be installed from PyPI or GitHub. The source code is maintained on GitHub.